HotFix/Utilities/Parser.cs

```diff
@@ -371,11 +393,21 @@ public static int WriteLong(this byte[] buffer, int position, long value)
             return position - initial;
         }
 
+        /// <summary>
+        /// Writes a double to the given byte array.
+        /// </summary>
+        /// <remarks>
+        /// The number is printed to 6 decimal places.
```

Comment:
- Can you put the remarks into the summary so that it shows up in intellisense?
- We should probably say `truncated` rather than `printed` - it's important to be explicit in this case